dom/test/node/mock-dom-source.ts

```diff
@@ -228,7 +230,7 @@ describe('isolation on MockedDOMSource', function() {
     }
 
     const {sinks, sources, run} = setup(app, {
-      DOM: () =>
+      DOM: (_: Stream<VNode>) =>
```

Comment:
Was this added to please TypeScript? I think we should still support "ignored main() sinks" so the inference is not strict about all main() sinks needing to have a respective sink-receiving driver. A change in tests indicates a possible breaking change too

Reply:
Are you sure? I would want a type error if a driver would ignore the sinks I return from main. This usually means i did something wrong

Reply:
Yes I'm quite sure this should be an allowed use case. Simplest scenario is taking a main() and testing only one channel of it. Or running only one of its channels. It improves the reusability of main() because it doesn't couple the main with any particular subset of sinks dictated by the drivers object.

Reply:
Isnt it the other way round? With this type the drivers object is dictated by the main function. Main is allowed to ignore drivers, but drivers are not allowed to ignore main

Reply:
Yes, drivers are allowed to ignore *some channels* from main.